cmd/utils.go

```diff
@@ -1391,6 +1396,17 @@ func GenDeploymentYaml(log *LoggingConfig, appName string, imageName string, con
 	}
 	return yamlFile, nil
 }
+
+//FindElement returns true if an element is in an array, and false otherwise
+func FindElement(slice []string, value string) bool {
+	for _, item := range slice {
+		if item == value {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
I believe we have a similar function already in utils?

Reply:
Thanks, yeah you're right. It was in the `testutils` file so I hadn't seen it. I moved it to the main utils so both files can use it. Also I agree on having it as a comma separated list, that's in the latest commit :)